app/src/main/java/com/poupa/vinylmusicplayer/service/MusicService.java

```diff
@@ -463,13 +461,8 @@ public int getPosition() {
         return playingQueue.getCurrentPosition();
     }
 
-    public void playNextSong(boolean force) {
-        if (force && PreferenceUtil.getInstance().maintainSkippedSongsPlaylist()) {
-            final long playlistId = MusicUtil.getOrCreateSkippedPlaylist(this).id;
-            PlaylistsUtil.addToPlaylist(this, getCurrentSong(), playlistId, true);
-        }
-
-        playSongAt(playingQueue.getNextPosition(force));
+    public void playNextSong(boolean skippedLast) {
+        playSongAt(playingQueue.getNextPosition(skippedLast), skippedLast);
```

Comment:
Can you renamed in function getNextPosition and getPreviousPosition the argument to skippedLast ?

Reply:
This will be the case on the next commit